### PR TITLE
Handling nulls on Bboxen.  Also a couple tests.

### PR DESCRIPTION
**Pull request by jiwalker-usgs**

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/model/Bbox.java [hunk 1]
```diff
@@ -62,14 +62,16 @@ public void setBbox(double minX, double minY, double maxX, double maxY) {
...
 		Envelope envelope = null;
-		Pattern pattern = Pattern.compile("BOX\\(\\s*([-\\d\\.]+)\\s+([-\\d\\.]+)\\s*,\\s*([-\\d\\.]+)\\s+([-\\d\\.]+)\\s*\\)");
-		Matcher matcher = pattern.matcher(bbox);
-		if (matcher.matches()) {
-			double minX = Double.parseDouble(matcher.group(1));
-			double minY = Double.parseDouble(matcher.group(2));
-			double maxX = Double.parseDouble(matcher.group(3));
-			double maxY = Double.parseDouble(matcher.group(4));
-			envelope = new Envelope(minX, maxX, minY, maxY);
+		if (bbox != null) {
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/model/Bbox.java]: So how is this handled upstream if null comes out of this?

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/gson/adapter/BboxAdapter.java [hunk 2]
```diff
@@ -11,49 +11,51 @@
...
+			JsonPrimitive minx = new JsonPrimitive(envelope.getMinX());
+			JsonPrimitive miny = new JsonPrimitive(envelope.getMinY());
+			JsonPrimitive maxx = new JsonPrimitive(envelope.getMaxX());
+			JsonPrimitive maxy = new JsonPrimitive(envelope.getMaxY());
+			bboxArray.add(minx);
+			bboxArray.add(miny);
+			bboxArray.add(maxx);
+			bboxArray.add(maxy);
+		}
+		return bboxArray;
```
Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/gson/adapter/BboxAdapter.java]: Any issues on handling a null upstream of this?

Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/gson/adapter/BboxAdapter.java]: This isn't null, just empty.  There isn't much else I can do with it if there isn't anything to serialize.

Comment (isuftin) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/gson/adapter/BboxAdapter.java]: Makes sense. :+1:

**Pull request merged**